icinga2/marid/scripts/icingaActionExecutor.groovy

```diff
@@ -19,9 +20,18 @@ alertFromOpsgenie = opsgenie.getAlert(alertId: alert.alertId)
 try{
     if (alertFromOpsgenie.size() > 0) {
 
+	def tz = TimeZone.getDefault()
+	def timestamp = new Date()
+
+	use ( TimeCategory ) {
+        	timestamp = timestamp + 30.minutes
```

Comment:
This 30.minutes can be a variable taken from Web interface, where user can set it to some default value one likes.